packages/flutter/lib/src/cupertino/sheet.dart

```diff
@@ -11,13 +11,18 @@ import 'interface_level.dart';
 import 'route.dart';
 import 'theme.dart';
 
+// The distance from the top of the open sheet to the top of the screen, as a ratio
+// of the total height of the screen. Found from eyeballing a simulator running
+// iOS 18.0.
```

Comment:
Nit: Have you tried this PR out on iPad? If not and you have time maybe give it a shot, just to double check that this proportion still looks ok on the bigger screen.

Reply:
It looks ok. Natively sheets adapt to a modal type view though. So there's nothing to compare against.
<img width="637" alt="Screenshot 2025-01-31 at 2 15 12 PM" src="https://github.com/user-attachments/assets/49d991e2-0774-4e5e-b4fa-223956d18ab4" />

Reply:
Oh I didn't realize that. Should we consider doing something similar?

Reply:
I'll create an issue for it, but I think it's going to be p3.